plugins/mumble_positional_audio_utils.h

```diff
@@ -18,8 +18,6 @@
 #endif
 
 #ifdef _MSC_VER
-#	include <corecrt_math_defines.h>
-#	include <intrin.h>
 #endif
```

Comment:
Let's remove the entire block.

Reply:
So the other include isn't needed either?

Reply:
Oh lol I just realized that I accidentally removed the second commit as well. But the CI is happy so I guess it is indeed unneeded xD